src/test/java/edu/hawaii/its/api/service/TestGrouperApiService.java

```diff
@@ -193,10 +193,8 @@ public void getSubject() {
 
         subjectsResults = grouperService.getSubjects("invalid-identifier");
         subjects = subjectsResults.getSubjects();
-        assertEquals(1, subjects.size());
-        subject = subjects.get(0);
-        assertNotNull(subject);
-        assertEquals("SUBJECT_NOT_FOUND", subject.getResultCode());
```

Comment:
It be should investigated why we no longer get the resultCode SUBJECT_NOT_FOUND. It would definitely be preferred to know which subjects were not found rather than removing them from the response.